tests/Unit/API/WP/OAuthServiceTest.php

```diff
@@ -3,9 +3,14 @@
 
 namespace Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\API\WP;
 
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Ads;
```

Comment:
Unused dependency

Reply:
Fixed here thx. https://github.com/woocommerce/google-listings-and-ads/pull/2424/commits/436881c453d613fce326d5599a38cd33148b7850